vioscsi/vioscsi.h

```diff
@@ -46,9 +46,26 @@ typedef struct VirtIOBufferDescriptor VIO_SG, *PVIO_SG;
 #define VIRTIO_SCSI_SENSE_SIZE 96
 
 #ifndef NTDDI_WINTHRESHOLD
-#define NTDDI_WINTHRESHOLD                  0x0A000000  /* ABRACADABRA_THRESHOLD */
+#define NTDDI_WINTHRESHOLD      0x0A000000  /* ABRACADABRA_THRESHOLD */
 #endif
 
+#ifndef NTDDI_WIN10_NI
+#define NTDDI_WIN10_NI          0x0A00000C
+#define NTDDI_WIN10_CU          0x0A00000D
+#endif
+
+#ifndef NTDDI_WIN11
+#define NTDDI_WIN11             NTDDI_WIN10_CO
```

Comment:
I think OS version defines should go to a separate file or to osdep.h

Reply:
@YanVugenfirer 
Can do, but it's worth mentioning this is historically where `ABRACADABRA_THRESHOLD` has been defined.
If you prefer, I can implement it in, say `ntddi_ver.h`...
Let me know.

Reply:
Will extract all NTDDI version definitions to `ntddi_ver.h`...